session_items.py

```diff
@@ -13,7 +13,9 @@ def get_items():
     Returns:
         list: The list of saved items.
     """
-    return session.get('items', _DEFAULT_ITEMS)
+    # return session.get('items', sorted(_DEFAULT_ITEMS, key=lambda item: item['status'], reverse=True))
+    _SORTED_ITEMS = _DEFAULT_ITEMS.sort(key = lambda item : item['status'], reverse=True)
```

Comment:
This only sorts the default items (i.e. before anything is added to the flask session cookie)

Reply:
Moved sort function to app.py